core/src/main/java/org/apache/iceberg/actions/BinPackStrategy.java

```diff
@@ -259,7 +259,7 @@ private void validateOptions() {
         MIN_FILE_SIZE_BYTES, RewriteDataFiles.TARGET_FILE_SIZE_BYTES, minFileSize, targetFileSize);
 
     Preconditions.checkArgument(targetFileSize < maxFileSize,
-        "Cannot set %s is greater than or equal to %s, all files written will be larger than the threshold, %d >= %d",
```

Comment:
I think the correct fix is to reverse `MAX_FILE_SIZE_BYTES` and `RewriteDataFiles.TARGET_FILE_SIZE_BYTES`. Leaving "or equal to" doesn't work.

Reply:
If  MAX_FILE_SIZE_BYTES equals to TARGET_FILE_SIZE_BYTES, argument check still works and throws this exception. Anyway, there is a mistake I forget to fix "%d >= %d" with parameters, I will reverse targetFileSize and maxFileSize later.
thanks for your comment

Reply:
I think this is incorrect for the equals case. Can you revert the message changes and swap the argument order?

Reply:
OK, I will follow your advice, swap the argument order.